src/main/java/com/epam/preprod/sirenko/DAO/ProductsInDBDAO.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.preprod.sirenko.DAO;
+
+import com.epam.preprod.sirenko.entity.Clothing;
+import com.epam.preprod.sirenko.entity.Food;
+import com.epam.preprod.sirenko.entity.Product;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductsInDBDAO {
+	Food food;
```

Comment:
We dont need separate product fields here

Reply:
changed